build/download-saturn-l2.js

```diff
@@ -67,12 +68,17 @@ async function main () {
           await pipeline(res.body, gunzip(), tar.extract(outFile))
         } else {
           // Darwin needs to be a zip for notarization
-          await mkdir(path.join(outDir, 'l2node-darwin-x64'), { recursive: true })
+          await mkdir(
+            path.join(outDir, 'l2node-darwin-x64'),
+            { recursive: true }
+          )
           const parser = unzip.Parse()
           await Promise.all([
             (async () => {
               while (true) {
-                const [entry] = /** @type {[UnzipStreamEntry]} */ (await once(parser, 'entry'))
+                const [entry] =
+                  /** @type {[UnzipStreamEntry]} */
+                  await once(parser, 'entry')
```

Comment:
Are you sure you can remove those `()` characters? I think they are a necessary part of the syntax to tell TypeScript to cast the value to a different type.

How do you feel about this variant?

```js
                const [entry] = /** @type {[UnzipStreamEntry]} */ (
                  await once(parser, 'entry')
                )
```